variants/vueapp/cypress/e2e/jsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Do we really want cypress? Can't we do it with jest (I'm using jest in my upcoming vue tests and Mikko is using this for sodar-server, I believe). Maybe remove this for now?

Reply:
OK, was added by default